mlrun/runtimes/base.py

```diff
@@ -391,6 +391,7 @@ def _get_db_run(self, task: RunObject = None):
         if task:
             return task.to_dict()
 
+    # TODO: remove this method
```

Comment:
so remove
or tell when i can be removed

Reply:
left it as a comment to myself to not forget (did it in a few places)